test/scripts/hpa-matrix-mult.lua

```diff
@@ -0,0 +1,129 @@
+local hpa = HPA()
+
+local function setup(np, dim)
+	local t0 = os.clock()
+
+	local csA = CellularSpace {
+		xdim = dim
+	}
+
+	local csB = CellularSpace {
+		xdim = dim
+	}
+
+	function fillCsA()
+		forEachCell(csA, function(cell)
+			cell.value = dim
+		end)
+	end
+
+	function fillCsB()
+		forEachCell(csB, function(cell)
```

Comment:
I was thinking about having a `forEachParallelCell()`. Would that be possible?

Reply:
I think so, but it doesn't work with a cellular space as an argument.

Reply:
Why not?

Reply:
Probably something that needs to be done in C++ level.